### ISSUE TITLE
integrate API Doc (reference) with agoric-sdk api-doc

### ISSUE THREAD

**Issue by katelynsills** [Agoric/documentation#319]
a possible plan:

 - [ ] choose domain name
   - note: the precendent used by endo is complicated by papers.agoric.com
 - [ ] publish docs there (continuously, on each change)
 - [ ] reconcile with API content here
 - [ ] integrate with search (see also #927)

earlier:
----

It appears that our types in agoric-sdk will cause jsDoc to error. For instance:

```sh
ERROR: Unable to parse a tag's type expression for source file /Users/katesills/code/agoric-sdk/packages/zoe/src/contractFacet/types.js in line 4 with tag title "property" and text "{(keyword: Keyword) => void} assertUniqueKeyword - check
whether a keyword is valid and unique and could be added in
`saveIssuer`": Invalid type expression "(keyword: Keyword) => void": Expected "$", "'", ".", "0", "@", "\"", "\\", "_", "break", "case", "catch", "class", "const", "continue", "debugger", "default", "delete", "do", "else", "enum", "export", "extends", "finally", "for", "function", "if", "implements", "import", "in", "instanceof", "interface", "let", "new", "package", "private", "protected", "public", "return", "static", "super", "switch", "this", "throw", "try", "typeof", "var", "void", "while", "with", "yield", Unicode letter number, Unicode lowercase letter, Unicode modifier letter, Unicode other letter, Unicode titlecase letter, Unicode uppercase letter, or [1-9] but " " found.
```

[TypeDoc](http://typedoc.org/) does not error and seems the most promising. It produces documentation like this:

<img width="894" alt="Screen Shot 2020-09-25 at 3 47 41 PM" src="https://user-images.githubusercontent.com/2441069/94323069-41366200-ff49-11ea-97d4-d6fd01e286b7.png">

It seems like it will take some configuration changes to make it usable. Then there is the challenge of reconciling it with our current site.

**Comment by dckc** [comment]
JSDoc errors were addressed a of:

 - https://github.com/Agoric/agoric-sdk/pull/8343

So we've done the exploration. I'm re-scoping this to integrate the docs site with the generated api-doc.

**Comment by sufyaankhan** [comment]
Dupe of 927

**Comment by dckc** [comment]
fixed in #1038
